src/Icons/src/IconRenderer.php

```diff
@@ -36,11 +42,19 @@ public function __construct(
      */
     public function renderIcon(string $name, array $attributes = []): string
     {
-        $name = $this->iconAliases[$name] ?? $name;
+        $iconName = $this->iconAliases[$name] ?? $name;
+
+        $icon = $this->registry->get($iconName)
+            ->withAttributes($this->defaultIconAttributes);
+
+        if (str_contains($name, ':')) {
+            $prefix = substr($name, 0, strpos($name, ':'));
+            if (isset($this->iconSetsAttributes[$prefix])) {
+                $icon = $icon->withAttributes($this->iconSetsAttributes[$prefix]);
+            }
```

Comment:
I think it can simplified to: 
```suggestion
            $attributes = ($this->iconSetsAttributes[$prefix] ?? []) + $attributes;
```

Reply:
I don't know how it's done internally indeed, but the "hot path" is "there is no $prefix key in $iconsSetAttributes[].. does this change something ?

Reply:
With my suggestion we can get rid of 2 functions calls and 1 access to `$iconSetsAttributes` property, and 1 access to `$this->iconSetsAttributes` array.

If you compare opcode (see https://3v4l.org/TAQmE/vld vs https://3v4l.org/5QdBE/vld), we have 4 less operations.

Reply:
No of course, i was only comparing isset and [] ?? []

I'll update the code (but i think i'll do it on "default + iconSet"  instead of "iconSet + attribute"

Reply:
This should be better now